src/site/stages/build/drupal/graphql/faqMultipleQa.graphql.js

```diff
@@ -62,6 +62,22 @@ fragment faqMultipleQA on NodeFaqMultipleQA {
       }
     }
   }
+  fieldTags {
+    entity {
+      ... on ParagraphAudienceTopics {
+        fieldTopics {
+          entity {
+            ... on TaxonomyTermTopics {
+              entityUrl {
+                path
+              }
+              name
+            }
+          }
+        }
+      }
+    }
+  }
```

Comment:
`fieldTags` is already part of this query on line 97. We just need to add `entityUrl` to `src/site/stages/build/drupal/graphql/taxonomy-fragments/termTopics.taxonomy.graphql.js`